Upstream-Status:Inappropriate [Backport]

Imported from perl git tree by Nitin A Kamble <nitin.a.kamble@intel.com>
2011-02-23

commit 6695a346c41138df5b2c0e26b9a49b1f96137da0
Author: Tony Cook <tony@openbsd32.tony.develop-help.com>
Date:   Thu Jul 22 09:54:13 2010 +1000

    make_ext.pl populates @INC correctly, don't override it badly
    
    PERL5LIB is populated by make_ext.pl with paths to the modules we need
    to run, don't override this with "../../lib" since that may not have
    been populated yet in a parallel build.

diff --git a/ext/POSIX/Makefile.PL b/ext/POSIX/Makefile.PL
index 392b6fb..9e6d091 100644
--- a/ext/POSIX/Makefile.PL
+++ b/ext/POSIX/Makefile.PL
@@ -1,7 +1,3 @@
-# Explicitly avoid including '.' in @INC; autoloader gets confused since it
-# can find POSIX.pm, but can't find autosplit.ix.
-BEGIN { @INC = '../../lib';}
-#
 use ExtUtils::MakeMaker;
 use ExtUtils::Constant 0.11 'WriteConstants';
 use Config;
